File: Uygulama12/MC/mcfrontend/lib/models/music.dart
class Music {
  int? id;
  String? name;
  int? year;
  Music.fromJson(Map<String, dynamic> json) {
    id = json["id"];
    name = json["name"];
    year = json["year"];
  }
  Music(this.name, this.year);
  Map<String, dynamic> toJson() {
    return {"id": id, "name": name, "year": year};
  }
}

File: Uygulama12/MC/mcbackend/BenzerBul/src/main/java/com/emirozturk/benzerbul/Model/Music.java
package com.emirozturk.benzerbul.Model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Entity
@Table(name = "Music")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Music {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id;
    @Column(name = "name")
    String name;
    @Column(name = "year")
    int year;
    @JsonIgnore
    @ManyToMany(mappedBy = "musics")
    List<User> Users;

    @Override
    public String toString() {
        return "Music{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", year=" + year +
                '}';
    }
}
